# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../subversion/sasl2-fix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur subversion-1.5.1-orig/subversion/include/private/ra_svn_sasl.h subversion-1.5.1/subversion/include/private/ra_svn_sasl.h
--- subversion-1.5.1-orig/subversion/include/private/ra_svn_sasl.h	2008-08-06 10:40:38.000000000 +0000
+++ subversion-1.5.1/subversion/include/private/ra_svn_sasl.h	2008-08-06 10:42:16.000000000 +0000
@@ -28,7 +28,7 @@
 #define STRUCT_IOVEC_DEFINED
 #include <sasl.h>
 #else
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 #endif
 
 #ifdef __cplusplus
